src/test/java/tech/aluvesoftware/steps/BaseTest.java

```diff
@@ -0,0 +1,29 @@
+package tech.aluvesoftware.steps;
+
+import io.restassured.response.Response;
+import lombok.Getter;
+import tech.aluvesoftware.api.http.Request;
+public class BaseTest {
+
+    @Getter
+    private static Request request;
+    @Getter
+    private Response response;
+
+    public void BaseUrlIs(String baseUrl) {
+        request = new Request(baseUrl);
+        request.build();
+    }
+
+    public void UserClicksOnTheButton(String endpoint, String http) {
```

Comment:
This is a api call/request, so they won't be any clicking. Please rename this.

Separation of concerns, if this is a Test Class, it shouldn't be having a method like this. All methods in a Test Class, we expect them to have an assert